lib/rules/jsx-sort-props.js

```diff
@@ -14,6 +14,19 @@ function isCallbackPropName(name) {
   return /^on[A-Z]/.test(name);
 }
 
+var RESEVED_PROPS_LIST = [
+  'children',
+  'dangerouslySetInnerHTML',
+  'htmlFor',
+  'key',
+  'ref',
+  'style'
+];
+
+function isReservedPropName(name, list) {
```

Comment:
(Part of making this trivial functionality a function was just to match the style of there also being an `isCallbackPropName` function.)